### DISCUSSION TITLE
Web console IP addresses

### DISCUSSION THREAD

**Discussion by partydrone** [nickjj/docker-rails-example#11] [Ideas]
@nickjj, I was actually about to submit a PR when I saw you added this. I will say that this is what I set in my **development.rb** file:

```ruby
config.web_console.allowed_ips = %w(10.0.0.0/8 172.16.0.0/12 192.168.0.0/16)
```

This covers all private networks, and I found that Docker installations don't always default to the same one. Mine, for example, defaulted to 192.168.0.0/16. You may want to consider updating your config to use all of them to cover more bases.

**Comment by nickjj** [comment]
Are you using Docker Toolbox? I've never seen 192.x with Docker Desktop.

I wonder if we should just use `0.0.0.0/0` and hard code it. That'll support everything and this isn't a case where it's less secure than using a local IP range because it's only applied to `development.rb`. Our puma bind address is already set to `0.0.0.0` too.

**Reply by partydrone** [reply]
I Dockerized a large project for a team at a company I worked for. Dev environments ran the gamut: macOS, Windows, and Linux. I use Docker Desktop for macOS, myself.

Hard coding `0.0.0.0/0` certainly simplifies things.

**Reply by nickjj** [reply]
Ok, I pushed the update. It gets rid of the env variable and sets `config.web_console.allowed_ips = ['0.0.0.0/0']` in `config/environments/development.rb`.

Here's the commit with the change: https://github.com/nickjj/docker-rails-example/commit/204fd1b9f20c15df67d1fa83b4e3a76ca368e99e
